CRM/Report/Form/ActivitySummary.php

```diff
@@ -20,6 +20,7 @@ class CRM_Report_Form_ActivitySummary extends CRM_Report_Form {
   protected $_phoneField = FALSE;
   protected $_tempTableName;
   protected $_tempDurationSumTableName;
+  protected $_totalRows;
```

Comment:
My understanding is the leading underscore is something of a legacy pattern & we have been ditching it over time